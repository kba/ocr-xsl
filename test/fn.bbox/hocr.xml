../samples/hocr/1.1/417576986_0013.hocr