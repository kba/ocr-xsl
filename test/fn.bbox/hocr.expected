<bbox>
   <left>140</left>
   <right>836</right>
   <top>253</top>
   <bottom>1782</bottom>
   <width>696</width>
   <height>1529</height>
</bbox>
